server/pages/index.js

```diff
@@ -391,12 +394,16 @@ module.exports = function() {
 				: 'all';
 
 			if ( config.isEnabled( 'server-side-rendering' ) ) {
+				const store = createReduxStore();
+				store.dispatch( setSection( 'design', { hasSidebar: false } ) );
+				context.store = pick( store.getState(), 'ui' );
```

Comment:
What looks a little suspicious to me tho is that we're assigning a `state` to a var named (and used as) `store`.